### PR TITLE
test/feat: Turn valid input length and character methods

### PR DESCRIPTION
**Pull request by Sergio-Azcona**

### CODE REVIEW

File: lib/turn.rb [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+    @player2 = player2
+    @board = board
   end
 
-
+  def valid_input_length(move)
+    move.length == 1
+  end
+  
+  def valid_input_character(letter)
```
Comment (Freeing3092) [inline lib/turn.rb:13]: Similar to above, can we add a '?' to the end of the method as the return value is boolean?

**Review by Freeing3092 — APPROVED** [review]
It looks like the tests are very thorough for considering edge cases for invalid user inputs.

My only comments were for minor naming conventions on the methods which can be refactored later.
